src/libs/shouldRenderAppPaymentCard/index.native.ts

```diff
@@ -0,0 +1,5 @@
+type ShouldRenderAddPaymentCard = () => boolean;
```

Comment:
If we duplicate this type twice, there is an option to create it in another file to import to both files. Also, do you think this type is necessary? Why not just const `shouldRenderAddPaymentCard = () => false;`

Reply:
@pac-guerreiro Please extract `ShouldRenderAddPaymentCard` type to a `types.ts` file under the same directory according to our TS guidelines.